aws/s3_test.go

```diff
@@ -321,7 +322,7 @@ type TestNukeS3BucketArgs struct {
 
 // testNukeS3Bucket - generates the test function for TestNukeS3Bucket
 func testNukeS3Bucket(t *testing.T, args TestNukeS3BucketArgs) {
-	awsParams, err := newS3TestAWSParams("")
+	awsParams, err := newS3TestAWSParams("eu-central-1")
```

Comment:
I was wondering why is required to use specific region

Reply:
@denis256 my organization has a SCP in place so i have only access to Resources in eu-central-1. This should not be a problem for the test with S3 buckets. If it is absolutely necessary, I can repeat the test again in a private account.

Thanks